src/lib/inbox/inbox.ts

```diff
@@ -276,4 +306,74 @@ export class InboxTools {
       overrides || {}
     )
   }
+
+  /**
+   * Send l2 signed tx using delayed inox, which won't alias the sender's adddress
+   * It will auto-confirm on l2, if it doesn't confirm in 24 hours, you can force
+   * include it.
+   * @param message A signed transaction which can be sent directly to network, 
+   *                you can call inboxTools.signL2Message to get.
+   * @returns The l1 delayed inbox's transaction receipt.
```

Comment:
This doesnt return the transaction receipt, just the transaction itself i think. They will need to call wait on it after to get the receipt.